packages/frameworks/src/frameworks.ts

```diff
@@ -1581,6 +1581,59 @@ export const frameworks = [
     buildCommand: 'vite build',
     getOutputDirName: async () => 'dist',
   },
+  {
+    name: 'Astro',
+    slug: 'astro',
+    demo: 'https://astro.examples.vercel.com',
+    logo: 'https://raw.githubusercontent.com/vercel/vercel/main/packages/frameworks/logos/astro.svg',
+    tagline:
+      'Astro is a next-generation static site generator that mixes lightning-fast performance with modern developer experience.',
+    description: 'An Astro site, using the default starter kit.',
+    website: 'https://astro.build',
+    envPrefix: 'SNOWPACK_PUBLIC_',
+    detectors: {
+      every: [
+        {
+          path: 'package.json',
+          matchContent:
+            '"(dev)?(d|D)ependencies":\\s*{[^}]*"astro":\\s*".+?"[^}]*}',
+        },
+      ],
+    },
+    settings: {
+      installCommand: {
+        placeholder: '`yarn install` or `npm install`',
+      },
+      buildCommand: {
+        placeholder: '`npm run build` or `astro build`',
+      },
+      devCommand: {
+        value: 'astro dev',
+      },
+      outputDirectory: {
+        value: 'dist',
+      },
+    },
+    dependency: 'astro',
+    devCommand: 'astro dev',
+    buildCommand: 'astro build',
+    getOutputDirName: async () => 'dist',
+    cachePattern: '.snowpack/**',
```

Comment:
@fredkschott can you verify this is a cache folder?

Reply:
I'm not sure what this config is for, but I know that Vite has a similar cache folder and they don't have a `cachePattern` key in their config.

Reply:
Okay, I noticed it in the `.gitignore` wasn't sure where it came from but I'll remove it for now. The `cachePattern` is to optimize build caches I believe. 

> A glob string of files to cache for future deployments